File: GCD_recursion.c
#include <stdio.h>
long int GCD(int a, int b)
{
    if (b == 0)
    {
        return a;
    }
    else
    {
        return GCD(b, a % b);
    }
}
int main()
{
    int a, b;
    printf("Enter the value of a and b :");
    scanf("%d%d", &a, &b);
    printf("The GCD result is : %ld", GCD(a, b));

    return 0;
}

File: Recursion/GCD_recursion.py
def GCD(a,b):
    if(b==0):
        return a
    else:
        return GCD(b,a%b)
    
a=int(input("Enter the value of a: "))
b=int(input("Enter the value of b: "))
c=GCD(a,b)
print(f'The GCD result using recursion is : {c}')
